env_pkmn.py

```diff
@@ -34,7 +34,10 @@ def scrape_input(self):
             # The action space line will still get transferred in the response.
             # There's no good reason for this beyond ''well it can't hurt''.
             if 'actionspace' in temp:
-                self.action_space = json.loads(temp[11:])
+                temp2 = json.loads(temp[11:])[0]
```

Comment:
It'd be nice if these temp variables had names that hinted more at what they are. If they don't represent anything meaningful, I guess it's ok, but this part of the code is bit opaque.

Reply:
Done